content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -330,6 +330,14 @@ Executors](https://argoproj.github.io/argo/workflow-executors/) and reference
     ```
 
     The Kubeflow Pipelines deployment may take several minutes to complete.
+    2. Verify that the Kubeflow Pipelines UI is accessible by port-forwarding:
+
+   ```SHELL
+   kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+   ```
+
+Open the Kubeflow Pipelines UI at http://localhost:8080/ or if you are using Kind or K3s within a VM http://{VM IP ADDRESS}:8080/
+Please note that K3ai will automatically print the right UI URL at the end of the installation process.
```

Comment:
Thanks! Suggestions @alfsuse :

- Add a space/separator, before step 2
- Use [code font](https://developers.google.com/style/code-in-text) for URLs (i.e. backticks)
- Some refactoring

```suggestion

2. Verify that the Kubeflow Pipelines UI is accessible by port-forwarding:

    ```shell
    kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
    ```

    Then, open the Kubeflow Pipelines UI at `http://localhost:8080/` or - if you are
    using kind or K3s within a virtual machine - `http://{YOUR_VM_IP_ADDRESS}:8080/`
    
    Note that K3ai will automatically print the URL for the web UI at the end of
    the installation process.
```

LMKWYT